Add link to Ethereum vision page (#7087)
ethereum/ethereum-org-website: src/content/energy-consumption/index.md

# Ethereum energy consumption {#introduction}

Ethereum's energy consumption will be reduced by ~99.95% following [The Merge](/upgrades/merge) from proof-of-work (PoW) to proof-of-stake (PoS). After The Merge, Ethereum will use dramatically less carbon to be more secure.

> Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development ([more on the Ethereum vision](/upgrades/vision/). Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that energy has been expended by the miner, demonstrating that they invested real-world value for the right to add to the blockchain. Both proof-of-work and proof-of-stake are just mechanisms to decide who gets to add the next block. Swapping proof-of-work for proof-of-stake, where the real-world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.

Reviewer: How about this @samajammin?

```suggestion
Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without comprising [Ethereum's vision of being a secure, scalable, and decentralized blockchain](/upgrades/vision/) has taken years of focused research and development. Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that energy has been expended by the miner, demonstrating that they invested real-world value for the right to add to the blockchain. Both proof-of-work and proof-of-stake are just mechanisms to decide who gets to add the next block. Swapping proof-of-work for proof-of-stake, where the real-world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
```

Reviewer 2: ```suggestion
Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without compromisng [Ethereum's vision of being a secure, scalable, and decentralized blockchain](/upgrades/vision/) has taken years of focused research and development. Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that energy has been expended by the miner, demonstrating that they invested real-world value for the right to add to the blockchain. Both proof-of-work and proof-of-stake are just mechanisms to decide who gets to add the next block. Swapping proof-of-work for proof-of-stake, where the real-world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
```

I like the suggestion, but think this was what you were going for with spelling.
